www/app/app.ts

```diff
@@ -204,7 +204,16 @@ export class App {
       return;
     }
     // Expect SHADOWSOCKS_URI.parse to throw on invalid access key; propagate any exception.
-    const shadowsocksConfig = SHADOWSOCKS_URI.parse(accessKey);
+    let shadowsocksConfig = null;
+    try {
+      shadowsocksConfig = SHADOWSOCKS_URI.parse(accessKey);
+    } catch (error) {
+      // Remove any access keys from the error message so it is not logged to Sentry.
+      const message = !!error.message ?
```

Comment:
Is this the only place in the client where we try to parse an `ss://` URL? This is okay as a quick fix but stripping IP addresses is **really** hard to get right - I'd much prefer we change Shadowsocks config not to ever include the original IP in its error messages.

Reply:
It is. Once I update ShadowsocksConfig not to log the access key, I'll remove this stopgap measure.